eo-maven-plugin/src/main/java/org/eolang/maven/objectionary/OyHome.java

```diff
@@ -87,4 +89,25 @@ public Input get(final String name) throws FileNotFoundException {
         }
         return new InputOf(file);
     }
+
+    @Override
+    public boolean contains(final String name) {
+        boolean ret;
+        try {
+            final Path file = new Place(name).make(
+                this.home
+                    .resolve("pulled")
+                    .resolve(this.version),
+                "eo"
+            );
+            ret = file.toFile().exists();
```

Comment:
@MikhailLipanin I believe you can use either `Home.exists()` or `Files.exists()` instead

Reply:
done